libraryversions.toml

```diff
@@ -178,6 +175,11 @@ WINDOW_SIDECAR = "1.0.0-rc01"
 WORK = "2.10.0-alpha04"
 XR = "1.0.0-alpha01"
 
+# JetBrains overrides
```

Comment:
It is okay to move difference below, but it looks like a half measure:
- we have `COMPOSE_` packages overriden, so conflicts still exist
- we have the source of truth of the CMP versions in https://jetbrains.team/p/ui/repositories/compose-teamcity-config/files/main/.teamcity/compose/Library.kt, so having versions here is misleading, we don't use them in anyway

The issue is similar to `settings.gradle` issue. We can do the same:

1. Add this file in `mergeAOSP` ignoring.

2. Set the content to:
```
CORE_BUNDLE = "9999.0.0-SNAPSHOT"
CORE_URI = "9999.0.0-SNAPSHOT"
COMPOSE = "9999.0.0-SNAPSHOT"
COMPOSE_MATERIAL3 = "9999.0.0-SNAPSHOT"
COMPOSE_MATERIAL3_ADAPTIVE = "9999.0.0-SNAPSHOT"
COMPOSE_MATERIAL3_COMMON = "9999.0.0-SNAPSHOT"
LIFECYCLE = "9999.0.0-SNAPSHOT"
NAVIGATION = "9999.0.0-SNAPSHOT"
GRAPHICS_SHAPES = "9999.0.0-SNAPSHOT"
WINDOW = "9999.0.0-SNAPSHOT"

COMPOSE_ANIMATION = { group = "org.jetbrains.compose.animation", atomicGroupVersion = "versions.COMPOSE" }
COMPOSE_COMPILER = { group = "org.jetbrains.compose.compiler", atomicGroupVersion = "versions.COMPOSE" }
COMPOSE_DESKTOP = { group = "org.jetbrains.compose.desktop", atomicGroupVersion = "versions.COMPOSE" }
COMPOSE_FOUNDATION = { group = "org.jetbrains.compose.foundation", atomicGroupVersion = "versions.COMPOSE" }
...

[groups.LIFECYCLE_JETBRAINS]
...
```

If we decide to move CMP versioning here, we replace it by the real versions (it is a separate discussion)